src/utils/io.js

```diff
@@ -53,7 +53,11 @@ export const requestAll = () => {
 
 export const requestApp = (appId) => {
   return axios
-    .get(buildURL(`/apps/${appId}`))
+    .get(buildURL(`/apps/${appId}`), {
+      headers: {
+        'x-server-auth': 'hplovecraft',
+      },
+    })
```

Comment:
@scott-massey 
I didn't know about this repo. Great work!

btw we should avoid include this kind of information hardcoded